code/style.css

```diff
@@ -1,4 +1,220 @@
-html {
-  background: #0025ff;
-  color: #fff;
-}
+
+
+* {
+   box-sizing: border-box; 
+   margin: 0;
+   padding: 0;
+}
+
+body {
+    margin: 0;
+    font-family: 'Oswald', sans-serif;
+    background-color: #fffef5;
+    width: 100vw;
+   
+}
+/* header styling for mobile */
+

```

Comment:
I think another approach could be, if you start with mobile first and to make it as a default.